node/forest_libp2p/src/blocksync/message.rs

```diff
@@ -5,11 +5,13 @@ use forest_blocks::{Block, BlockHeader, FullTipset, Tipset, BLOCK_MESSAGE_LIMIT}
 use forest_cid::Cid;
 use forest_encoding::tuple::*;
 use forest_message::{SignedMessage, UnsignedMessage};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::convert::TryFrom;
 
 /// Blocksync request options
 pub const BLOCKS: u64 = 1;
 pub const MESSAGES: u64 = 2;
+pub const BLOCKS_MESSAGES: u64 = 3;
```

Comment:
Don't need this

Reply:
it's in a spec though?

Reply:
yeah it's not necessary though, that only exists because it wasn't bit flags before. `0b01 | 0b10 == 3`. There is no need to use this constant itself.

Reply:
> yeah it's not necessary though, that only exists because it wasn't bit flags before

Sorry, when you say `before` are you referring to your suggestion here: https://github.com/ChainSafe/forest/pull/724#discussion_r500290219, or it's bit flags in a spec now too?